mmcv/cnn/utils/weight_init.py

```diff
@@ -200,8 +199,8 @@ def init(m):
                     xavier_init(m, self.gain, self.bias, self.distribution)
 
         module.apply(init)
-        if hasattr(module, '_params_init_info'):
-            update_init_info(module, init_info=self._get_init_info())
+        if hasattr(module, '_update_init_info'):
```

Comment:
Currently, the logic here is much different from the early version. In this version, you check whether the `module` contains `_update_init_info` function. Will this behave differently from the last version?

Reply:
In this implementation, the `_update_init_info` and `_params_init_info` always appear or be deleted at the same time, and I check the existence of `_params_init_info` in `_update_init_info`. I believe it can keep the same behavior